provider/docker/docker.go

```diff
@@ -721,6 +725,45 @@ func (p *Provider) getBasicAuth(container dockerData) []string {
 	return []string{}
 }
 
+func (p *Provider) hasRequestHeaders(container dockerData) bool {
+	if _, err := getLabel(container, types.LabelFrontendRequestHeader); err != nil {
```

Comment:
Maybe you could evaluate the len?

```go
label, err := getLabel(container, types.LabelFrontendRequestHeader)
return err == nill && len(label) > 0
```

same for `hasResponseHeaders`

Reply:
👍